### PR TITLE
confluent kafka deployment changes

### PR DESCRIPTION
**Pull request by KPostOffice**
## Related Issues and Dependencies

https://github.com/thoth-station/messaging/issues/319

closes https://github.com/thoth-station/investigator/issues/346

## This introduces a breaking change

- [ ] Yes
- [x] No

## Test or Stage Environment

- [ ] Pull Requests added content to STAGE environment
- [ ] Pull Requests to `AICoE/aicoe-cd` has been opened: 

## This Pull Request implements

Change env variables to reflect confluent kafka implementation

### CODE REVIEW

File: investigator/overlays/test/kustomization.yaml [hunk 1]
```diff
@@ -7,6 +7,7 @@ resources:
   - route.yaml
   - service.yaml
   - thoth-notification.yaml
+  - deploymentconfig.yaml
```
Comment (harshad16) [inline investigator/overlays/test/kustomization.yaml:10]: Add this under the key patchesStrategicMerge, as this has to patch into the base dc.

```suggestion
patchesStrategicMerge:
  - deploymentconfig.yaml
```

**Review by harshad16 — COMMENTED** [review]
LGTM so far , just a minor comment.

File: investigator/overlays/test/kustomization.yaml (hunk 1 above)
Comment (KPostOffice) [reply investigator/overlays/test/kustomization.yaml:10]: Got it, ty

File: investigator/overlays/test/kustomization.yaml [hunk 2]
```diff
@@ -7,6 +7,8 @@ resources:
   - route.yaml
   - service.yaml
   - thoth-notification.yaml
+patchesStrategicMerge:
+    - "deploymentconfig.yaml"
```
Comment (harshad16) [inline investigator/overlays/test/kustomization.yaml:11]:
```suggestion
   - deploymentconfig.yaml
```

File: investigator/overlays/test/kustomization.yaml [hunk 3]
```diff
@@ -7,6 +7,8 @@ resources:
   - route.yaml
   - service.yaml
   - thoth-notification.yaml
+patchesStrategicMerge:
+   - deploymentconfig.yaml
```
Comment (harshad16) [inline investigator/overlays/test/kustomization.yaml:11]:
```suggestion
  - deploymentconfig.yaml
```

**Pull request merged**